examples/ale_classification.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<p>Ok, a bit confused here as there is no positive effect of 0.69 in that vector. Maybe we could be a bit more precise and say that all the effects are due to the combined feats except petal length.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>I changed the order around to be <code>mean_logits - subset_logits</code>  so the interpretation is more clear. Updated text to</p><p> </p><p>"</p><p>Now if we subtract the logits of the instances for which petal length is ~3.8 from the mean logits, because $\text{ALE}(3.8)\approx 0$ for petal length, any difference from zero must be due to the combined effect of all other features (except petal length):</p><p>"</p><p>...code...</p><p>"For example, the remaining 3 features combined must be responsible for a positive effect of around ~0.69 on predicting instances with petal length ~3.8cm to be of class <code>setosa</code>."</p>